src/bundle/Resources/public/js/CKEditor/link/link-ui.js

```diff
@@ -201,10 +201,20 @@ class IbexaLinkUI extends Plugin {
         });
     }
 
+    isLinkElement(element) {
+        return element.is('attributeElement') && !!element.hasAttribute('href');
+    }
+
     findLinkElement() {
+        const modelElement = this.editor.editing.view.document.selection.getSelectedElement();
```

Comment:
```suggestion
        const viewElement = this.editor.editing.view.document.selection.getSelectedElement();
```
this is view element, the question is do we want to rely on view element or model element?